src/Entity/Banner.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Baraja\Banner\Entity;
+
+
+use Baraja\Doctrine\Identifier\IdentifierUnsigned;
+use Baraja\Localization\TranslateObject;
+use Baraja\Localization\Translation;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity]
+#[ORM\Table(name: 'core__banner')]
+class Banner
+{
+	use IdentifierUnsigned;
+	use TranslateObject;
+
+	#[ORM\Column(type: 'translate')]
+	private Translation $name;
+
+	#[ORM\Column(type: 'string', length: 64, unique: true)]
+	private string $slug;
+
+	#[ORM\Column(type: 'string', length: 16)]
+	private string $type;
+
+	#[ORM\Column(type: 'text', nullable: true)]
+	private ?string $description = null;
+
+	#[ORM\Column(type: 'boolean')]
+	private bool $active = false;
+
+	#[ORM\Column(type: 'integer')]
+	private int $width;
+
+	#[ORM\Column(type: 'integer')]
+	private int $height;
+
+	#[ORM\Column(type: 'json')]
+	private array $meta = [];
+
+	/** @var BannerItem[]|Collection */
+	#[ORM\OneToMany(mappedBy: 'banner', targetEntity: BannerItem::class)]
+	private $bannerItems;
+
+
+	public function __construct(Translation $name, string $slug, string $type, int $width, int $height)
```

Comment:
Name bude jen string, nastavuj přes `$this->setName($name)` (magický setter, zajišťuje ho Translate vrstva).